test/suite/test_hs18.py

```diff
@@ -107,17 +107,20 @@ def test_base_scenario(self):
         self.session.commit_transaction('commit_timestamp=' + self.timestamp_str(10))
 
         # Evict the update using a debug cursor
-        cursor.reset()
         self.evict_key(uri)
 
         # Commit an update without a timestamp on our original key
         self.session.begin_transaction('no_timestamp=true')
-        cursor[self.create_key(1)] = value4
+        cursor.set_key(self.create_key(1))
```

Comment:
Suggestion: we could replace those 3*3 lines with a call to a function:
```
    def update_kv(self, cursor, key, value):
        cursor.set_key(key)
        cursor.set_value(value)
        cursor.update()
```

Reply:
Thanks. Fixed as per the suggestion to reduce the code duplication.